src/gobexport/app.py

```diff
@@ -168,14 +167,14 @@ def dump_on_new_events(msg):
 def run_message_thread():
     try:
         messagedriven_service(SERVICEDEFINITION, "Export")
-    except:  # noqa: E722 do not use bare 'except'
-        pass
-    print("ERROR: no connection with GOB message broker, application is stopped")
-    os._exit(os.EX_UNAVAILABLE)
+    except Exception as e:  # noqa: E722 do not use bare 'except'
+        print("ERROR: no connection with GOB message broker, application is stopped")
+        print(e)
+        os._exit(os.EX_UNAVAILABLE)
```

Comment:
Original code is always exits with os.exit, not only on exception.  Intentional change?

Reply:
nice catch, not tested though ;-) 100% coverage ftw!